amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -211,6 +213,21 @@ class TableDetail extends React.Component<TableDetailProps & RouteComponentProps
                   </EditableSection>
                 </section>
               </section>
+              {
+                Array.from(data.programmatic_descriptions.values())
+                  .map((d: ProgrammaticDescription) =>
```

Comment:
1. If `data.programmatic_descriptions` is an array, why not `data.programmatic_descriptions.map(...)`?

2. Do yo actually need to import `ProgrammaticDescription` and insert the type here? Does TypeScript not implicitly recognize `d`  is a `ProgrammaticDescription` because we declared the type for this array already in the `TableMetadata` interface?

Reply:
thanks, done